File: OOPS_LAB/Program_3/vector3d.java
package OOPS_LAB.Program_3;

class vector{
    private int x;
    private int y;
    private int z;

    public vector(int xip,int yip,int zip) {
        x=xip;
        y=yip;
        z=zip;
    }

    public void VectorMagnitude()
    {
        double sumOfSquare = Math.pow(this.x, 2)+Math.pow(this.y, 2)+Math.pow(this.z, 2);

        double ans = Math.sqrt(sumOfSquare);
        System.out.println("\nMagnitu of vector "+ans);
    }
}

public class vector3d {
    public static void main(String[] args) {
        vector vec1 = new vector(3, 4, 5);
        vec1.VectorMagnitude();
    }
}

File: OOPS_LAB/Program_3/vector3d.cpp
#include<iostream>
#include<cmath>
using namespace std;

class Vector3D{
    private:
        int x,y,z;
    public:

    Vector3D(int xIP,int yIP,int zIP)
    {
        x=xIP;
        y=yIP;
        z=zIP;
    }

    void VectorMagnitude()
    {
        double sumOfSquare=pow(x,2)+pow(y,2)+pow(z,2);
        double ans = sqrt(sumOfSquare);
        cout<<"Magnitude of vector : "<<ans;
    }
};

int main(int argc, char const *argv[])
{
    Vector3D *vec1 = new Vector3D(3,4,5);
    vec1->VectorMagnitude();
    delete vec1;
    return 0;
}
